### ISSUE TITLE
Calendar panel background colour in 0.110

### ISSUE THREAD

**Issue by gadgetchnnel** [JuanMTech/google_dark_theme#20]
In the 0.110 beta of Home Assistant, there is a new Calendar panel. The calendar background is displaying as white even when this theme is selected. Adding this to the theme appears to fix it:

```
card-background-color: var(--paper-card-bckground-color)
```

**Comment by tigattack** [comment]
I tried the change you suggested, though I am running 0.112.

### Before: 
![image](https://user-images.githubusercontent.com/10629864/86284155-3b471400-bbda-11ea-8672-b1235a1dd953.png)

### After:
![image](https://user-images.githubusercontent.com/10629864/86284163-3e420480-bbda-11ea-816a-b76a627f1252.png)

Broken either way 😞

**Comment by gadgetchnnel** [comment]
Do you have the latest version of this theme? I works fine for me, and all I have different from the latest version, and the only difference is the `card-background-color` part.

Since the theme bases all the card colours off paper-dialog-background-color, try it with the cards section of the theme like this:

```
  # Cards
  ha-card-border-radius: "10px"
  ha-card-box-shadow: 1px 1px 5px 0px rgb(12, 12, 14)
  paper-dialog-background-color: "#202124"
  paper-card-background-color: var(--paper-dialog-background-color)
  paper-listbox-background-color: var(--paper-dialog-background-color)
  card-background-color: var(--paper-dialog-background-color)
```

(The ordering may or make not make a difference, but probably best to have `card-background-color` defined after `paper-dialog-background-color` since it references it).

**Comment by JuanMTech** [comment]
Just fixed it on version 1.5
